general/administration/quick-start.md

```diff
@@ -3,15 +3,16 @@ uid: admin-quick-start
 title: Quick Start
 ---
 
-# Quick-Start
+# Quick Start
 
 1. [Install Jellyfin on your system](xref:admin-installing).
 
-2. Browse to `http://<yourmachine>:8096`
+2. Browse to `http://<yourmachine>:8096`.
```

Comment:
Can we recommend browsing to `http://127.0.0.1:8096` instead?

Reply:
Sounds good to me, I'll commit it here if you add the suggestion.

Reply:
You guys have an issue if I remove stuff like (xref:admin-installing) And put the actual url in a later PR? The xref work on the official docs but I typically work out of the git docs which doesn't support it lol

Reply:
I don't like the DocFX stuff much either, but since that's what we're using, we should use the included features. One last thing I haven't decided on a standard for is referencing server pages. Some of them are in bold but I think I just want to reference them in plain text.

Reply:
```suggestion
2. Browse to `http://127.0.0.1:8096`.
```

Reply:
Please don't commit "actual" URLs, since there is no such thing, the config can change these at any point almost trivially. Just use the `docfx serve` locally, support every feature you need.